frontend/src/pages/GraphicalSelector/GraphicalSelector.tsx

```diff
@@ -224,6 +226,10 @@ const GraphicalSelector = () => {
     graph.zoomTo(zoom - getZoomRatio());
   };
 
+  const toggleSidebar = () => {
+    setSidebar(!sidebar);
```

Comment:
```suggestion
    setSidebar(sidebar => !sidebar);
```
to make sure that all input is taken

Reply:
wait can you elaborate on this